### PR TITLE
Add an example github issue tagger

### PR DESCRIPTION
**Pull request by nkolev92**

### CODE REVIEW

File: GithubIssueTagger/Program.cs [hunk 1]
```diff
@@ -1,85 +1,15 @@
...
-                            Console.WriteLine($"Updated issue: {issue.HtmlUrl}");
-                        }
-                        catch (Exception e)
-                        {
-                            Console.WriteLine($"Unhandled issue {issue.HtmlUrl} {e}");
-                        }
-                    }
-                }
-            }
+            client.Credentials = new Credentials(args[0]);
```
Comment (zivkan) [inline GithubIssueTagger/Program.cs]: I know this is a non-production tool, but the most basic of arg checking would be nice :)

```cs
if (args.count != 1)
{
  Console.Err.WriteLine("Expected 1 argument (github PAT). Found " + args.Count);
  return;
}
```

File: GithubIssueTagger/PullRequestUtilities.cs [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                var importantStat = stats.Where(e => e.Value.State == "closed" && e.Value.MergedDate != null).OrderByDescending(e => e.Value.MergedDate)
+                    .Select(e => new Stat(e.Value.Creator, e.Key, e.Value.CreatedAt, (DateTimeOffset)e.Value.MergedDate, (DateTimeOffset)e.Value.GetFirstEngagementDate(), e.Value.GetReviewers())).ToArray();
+                File.WriteAllText(closedIssueFileName, JsonConvert.SerializeObject(importantStat, Formatting.Indented));
+            }
+
+            if (AnalyzeReviewDistribution)
+            {
+                var lines = File.ReadAllText(closedIssueFileName);
+                var stats = JsonConvert.DeserializeObject<Stat[]>(lines);
+                var interestingPeople = new string[] { "kartheekp-ms", "cristinamanum", "zivkan", "heng-liu", "nkolev92", "aortiz-msft", "dominoFire", "rrelyea", "dtivel", "donnie-msft", "zkat" };
```
Comment (zivkan) [inline GithubIssueTagger/PullRequestUtilities.cs]: I'm honoured! :)

File: GithubIssueTagger/Program.cs (hunk 1 above)
Comment (nkolev92) [reply GithubIssueTagger/Program.cs]: Add a suggestion and I'll merge it. 

I really don't want to be making changes :D

Comment (nkolev92) [reply GithubIssueTagger/Program.cs]: Added it.

**Pull request merged**